mno_analysis_tools/docker-run-compute-msg-difference-btwn-two-firebase-time-periods.sh

```diff
@@ -53,7 +56,7 @@ fi
 
 CMD="pipenv run $PROFILE_MEMORY_CMD python -u compute_msg_difference_btwn_two_firebase_time_periods.py \
     /data/raw_messages.json /data/${MSG_DIRECTION}_msg_diff_per_period.json \
-    \"$TARGET_OPERATOR\" \"$TARGET_MESSAGE_DIRECTION\"  \"$START_DATE\" \"$END_DATE\" -t \"$TIME_FRAME\"
+    \"$TARGET_OPERATOR\" \"$TARGET_MESSAGE_DIRECTION\"  \"$START_DATE\" \"$END_DATE\" ${TIME_FRAME_ARG}
```

Comment:
Flags usually get listed first (because some scripts only accept flags if they come first, like this shell script), i.e. python -u compute_msg_difference_btwn_two_firebase_time_periods.py ${TIME_FRAME_ARG} /data/raw_messages.json etc.

Reply:
Also, check if you need to "" TIME_FRAME_ARG or not, I can never remember with bash

Reply:
Yeah its possible. I've updated, thanks.